driver-mapping/src/main/java/com/datastax/driver/mapping/MethodMapper.java

```diff
@@ -92,6 +88,30 @@ public void prepare(MappingManager manager, PreparedStatement ps) {
         }
     }
 
+    // Checks the method parameters against the query's bind variables
+    private void validateParameters() {
+        if (method.isVarArgs())
+            throw new IllegalArgumentException(String.format("Invalid varargs method %s in @Accessor interface", method.getName()));
+
+        ColumnDefinitions variables = statement.getVariables();
+        Set<String> names = Sets.newHashSet();
```

Comment:
I thought about it, but `ColumnDefinitions` is in the core package, so we would have to change the public API. I prefer being conservative with that, and I don't see a strong case for `ColumnDefinitions#getNames()` in general.
In addition, this method is called when initializing an accessor so performance is not paramount here (and even then, creating this extra set would not make a big difference).